### DISCUSSION TITLE
中秋快乐

### DISCUSSION THREAD

**Discussion by zkitefly** [Meloong-Git/PCL#2728] [讨论]
虽然不是与 PCL 有关，但还是想说一个：中秋快乐🎑

https://github.com/Hex-Dragon/PCL2/assets/64117916/0d571411-0e56-4961-8220-76c92062be34

**Comment by iPodtouch6557** [comment]
无锡有烟花看，但是没拍，忙着烧火喝青岛啤酒呢（
这是视频

https://github.com/Hex-Dragon/PCL2/assets/118656946/cb9c8d30-66be-44a4-b9e6-7f0d228cbb1a

**Comment by MoYuan-CN** [comment]
中秋快乐~

**Reply by MoYuan-CN** [reply]
感觉最近帖子新增率下降了啊

**Reply by kong-que-dnf123** [reply]
估计是找不出什么bug或者优化了，也算是个好事（？）  **_（侧面说明PCL越发完美qwq）_**

**Reply by MoYuan-CN** [reply]
嗯

**Comment by SqrtTan** [comment]
此时的我正在为铁杆粉奋斗……

**Reply by SqrtTan** [reply]
当然，还有欧皇彩（

**Reply by MoYuan-CN** [reply]
啊哈
